modules/spo-memory-profiling.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="spo-memory-profiling_{context}"]
+= Enable CPU and memory profiling
+
+You can enable the CPU and memory profiling endpoints for debugging purposes.
+
+.Procedure
+
+. To use the profiling support, patch the spod config and set the `enableProfiling` value by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-security-profiles patch spod \
+spod --type=merge -p '{"spec":{"enableProfiling":true}}'
+----
++
+.Example output
+[source,terminal]
+----
+securityprofilesoperatordaemon.openshift-security-profiles.x-k8s.io/spod patched
+----
+
+. Verify the openshift-security-profiles container is serving the profile endpoint by running the following command:
++
+[source,terminal]
+----
+$ k logs --selector name=spod -c openshift-security-profiles | grep "Starting profiling"
```

Comment:
I'm assuming this is an alias for `kubectl` right? I can safely change this to `oc logs`?